src/Microsoft.ML.Featurizers/TimeSeriesImputer.cs

```diff
@@ -61,15 +61,7 @@ public static class TimeSeriesImputerExtensionClass
     }
 
     /// <summary>
-    /// Imputes missing rows and column data per grain, based on the dates in the date column. This operation needs to happen to every column in the IDataView,
-    /// If you "filter" a column using the filterColumns and filterMode parameters, if a row is imputed the default value for that type will be used.
-    /// Currently only float/double/string columns are supported for imputation strategies, and an empty string is considered "missing" for the
-    /// purpose of this estimator. A new column is added to the schema after this operation is run. The column is called "IsRowImputed" and is a
-    /// boolean value representing if the row was created as a result of this operation or not.
-    ///
-    /// NOTE: It is not recommended to chain this multiple times. If a column is filtered, the default value is placed when a row is imputed, and the
-    /// default value is not null. Thus any other TimeSeriesImputers will not be able to replace those values anymore causing essentially a very
-    /// computationally expensive NO-OP.
+    /// Imputes missing rows and column data per grain, based on the dates in the date column.
```

Comment:
Looking much better! What does "grain" mean in this context?

Reply:
Grain is essentially the category. Not sure why its called a "grain", but thats the terminology they use.

Reply:
Who are "they"? :-) Is the term relevant to the user of this API? If not, I would just change it to `category`